src/components/Sidebar.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <b-button v-b-toggle.sidebar-right>Related Movies</b-button>
+    <b-sidebar id="sidebar-right" title="Related Movies" right shadow>
+      <div class="px-3 py-2" v-for="movie in movies" :key="movie.id">
+        <b-card
+          :title="movie.title"
+          :img-src="movie.imageCover"
+          img-alt="Image"
+          img-top
+          tag="article"
+          style="max-width: 20rem"
+          img-width="200px"
+          img-height="250px"
+          class="mb-2 related"
+          @click="goToMovie(movie.id)"
+        >
+        </b-card>
+      </div>
+    </b-sidebar>
+  </div>
+</template>
+
+<script>
+import MovieService from "../services/MovieService";
+export default {
+  props: {
+    relatedBy: String,
+    movieId: Number,
+  },
+  data() {
+    return {
+      relatedMovies: [],
+    };
+  },
+  computed: {
+    movies() {
+      if (this.relatedMovies.data !== undefined) {
```

Comment:
hajde da ovo lepse napiemo :)  return !this.relatedMovies.data  ? [] : this.relatedMovies.data.filter((x) => x.id !== this.movieId);